src/CapabilityService.Tests/TestDoubles/StubCapabilityApplicationService.cs

```diff
@@ -9,12 +9,12 @@ namespace DFDS.CapabilityService.Tests.TestDoubles
 {

```

Comment:
We do have a lot of test doubles. I have worked a lot with mocking libraries and they can replace most test doubles easy. 
Some of the tradeoffs I experienced using mock libraries was:
- My unit tests was harder to comprehend because I needed to prepare a mock object. This could take many more lines than the rest of the test.
- I placed a mock object in place everywhere i needed a test double even when I needed a Dummy Object or a test stub.
- I duplicated a lot of stup setup between unit tests so I started to create mock factories, but it was hard to communicate what they produced so the just felt more and more messy.

Which have lead me to writing my own test doubles.

As you have pointed out writing our own test doubles have issues as well, so I am fine with anyone using a mock library. And I will raise a flag if we start seeing some of the issues I have faced with mock libraries.